### PR TITLE
Fixed Order of questions and few issues in table

### PR DESCRIPTION
**Pull request by mekalaka**

### CODE REVIEW

File: src/Traisi/Controllers/SurveyAnalyticsController.cs [hunk 1]
```diff
@@ -158,130 +150,181 @@ await this
...
+                                            .FirstOrDefault(x =>
+                                                x.Code ==
+                                                ((OptionSelectResponse) p).Code)
+                                            .Order
+                                };
+                            }
+                            else
+                            {
+                                return new {
+                                    Label = "NOTA",
```
Comment (BrendanReilly) [inline src/Traisi/Controllers/SurveyAnalyticsController.cs]: Maybe make NOTA string a global const (in traisi.data) or if you want to suggest a location

**Review by BrendanReilly — COMMENTED** [review]
Added note about making NOTA options constant in another class (Traisi.Data) -- to avoid hard coding strings.

File: src/Traisi/Controllers/SurveyAnalyticsController.cs (hunk 1 above)
Comment (mekalaka) [reply src/Traisi/Controllers/SurveyAnalyticsController.cs]: Made NOTA a global constant.

File: src/Traisi/Controllers/SurveyAnalyticsController.cs [hunk 2]
```diff
@@ -158,130 +150,181 @@ await this
...
                         .Where(item =>
                             item.SurveyResponse.QuestionPart.Id == questionId &&
                             item.SurveyResponse.QuestionPart.SurveyId ==
-                            surveyId).ToListAsync()).Select(p => new
+                            surveyId)
+                        .ToListAsync()
+                    )
+                        .Select(p =>
+                        {
+                            if (((OptionSelectResponse) p).Code != "nota")
```
Comment (BrendanReilly) [inline src/Traisi/Controllers/SurveyAnalyticsController.cs]: still a hard coded "nota"

**Review by BrendanReilly — CHANGES_REQUESTED** [review]
There is a hard coded nota that should be removed (maybe missed it?)

File: src/Traisi/Controllers/SurveyAnalyticsController.cs (hunk 2 above)
Comment (mekalaka) [reply src/Traisi/Controllers/SurveyAnalyticsController.cs]: Fixed the hard coded "nota" and pushed changes

**Review by BrendanReilly — APPROVED** [review]

**Pull request merged**